lib/ApiOperations/Request.php

```diff
@@ -32,15 +32,16 @@ protected static function _validateParams($params = null)
      * @param array $params list of parameters for the request
      * @param null|array|string $options
      * @param string[] $usage names of tracked behaviors associated with this request
+     * @param 'v1'|'v2' $apiMode
      *
      * @throws \Stripe\Exception\ApiErrorException if the request fails
      *
      * @return array tuple containing (the JSON response, $options)
      */
-    protected function _request($method, $url, $params = [], $options = null, $usage = [])
+    protected function _request($method, $url, $params = [], $options = null, $usage = [], $apiMode = 'v1')
```

Comment:
@ramya-stripe do we need apiMode to passed as a parameter? Can't we determine apiMode from URL like we do in other libraries and get rid of this extra parameter? Could be a followup.